webapp/src/components/PayMortgagePage/PayMortgagePage.container.js

```diff
@@ -0,0 +1,44 @@
+import { withRouter } from 'react-router'
+import { connect } from 'react-redux'
+import { push } from 'react-router-redux'
+
+import PayMortgagePage from './PayMortgagePage'
+import {
+  payMortgageRequest,
+  fetchActiveParcelMortgagesRequest
+} from 'modules/mortgage/actions'
+import { locations } from 'locations'
+import { getMatchParamsCoordinates } from 'modules/location/selectors'
+import {
+  getParcelMortgageFactory,
+  isFetchingParcelMortgages
+} from 'modules/mortgage/selectors'
+import { isLoading } from 'modules/parcels/selectors'
+import { getWallet, isConnected, isConnecting } from 'modules/wallet/selectors'
+
+const mapState = (state, ownProps) => {
+  const { x, y } = getMatchParamsCoordinates(ownProps)
+  const getMortgage = getParcelMortgageFactory(x, y)
+  return state => ({
+    x,
+    y,
```

Comment:
Same as `ParcelDetailPage` above, the `x,y` won't change if the reducer updates. Is this a problem here?

Reply:
It should not cause any problem here, no need for re-fetching